src/mixins/view.js

```diff
@@ -211,6 +212,16 @@ export default {
     return ret;
   },
 
+  // Cache `childViewEvents` and `childViewTriggers`
+  _buildEventProxies: function() {
+
+    var childViewEvents = this.getOption('childViewEvents');
+    this._childViewEvents = this.getValue(childViewEvents);
+
+    var childViewTriggers = this.getOption('childViewTriggers');
+    this._childViewTriggers = this.getValue(childViewTriggers);
```

Comment:
small knit, but I'd prefer to see `this.getValue(this.getOption('childViewTriggers'))` I think this is nicer, even w/o the util